Tests/src/InternalTests/SerialisationTests.cpp

```diff
@@ -55,35 +115,15 @@ CSP_INTERNAL_TEST(CSPEngine, SerialisationTests, SpaceEntityUserSignalRSerialisa
 	User->Serialise(Serialiser);
 	auto SerialisedUser = Serialiser.Finalise();
 
-	/*
```

Comment:
Why was this comment removed? It's useful for if future developers need to modify these tests.